### ISSUE TITLE
PropCheck.produce/1 fails

### ISSUE THREAD

**Issue by gasparch** [alfert/propcheck#26]
This code fails 

```
use PropCheck
produce(integer())
```
with error

```
** (MatchError) no match of right hand side value: :undefined
    (proper) src/proper.erl:638: :proper.global_state_init/1
    (proper) src/proper_gen.erl:210: :proper_gen.pick/3
```

versions
```
propcheck 1.0.3 (also HEAD of master) 
proper  1.2.0 
```

Seems this line is at fault 

https://github.com/alfert/propcheck/blob/2553e7880c8a16aadb697af62299ccff8f660a35/lib/propcheck.ex#L910

This works
```
produce(integer(), :os.timestamp())
```

**Comment by evnu** [comment]
Does PR #25 fix that for you?

**Comment by gasparch** [comment]
Yes, works for me.

**Comment by alfert** [comment]
Solved via PR #25, thanks for reporting!
